packages/ui/src/QueryProvider/index.tsx

```diff
@@ -9,7 +9,7 @@ type QueryProviderWrapperProps = {
 }
 
 export function QueryProvider({ children, persister, queryClient }: QueryProviderWrapperProps) {
-  if (persister) {
+  if (persister && !window?.localStorage?.getItem('react-query-no-persist')) {
```

Comment:
What's the reason? 🤔

Reply:
it's annoying to develop when all the hooks get cached, so when I change them they don't get called

Reply:
Try to implement devtools for that
https://tanstack.com/query/latest/docs/framework/react/devtools